lib/css_injector.js

```diff
@@ -14,14 +14,50 @@ CSSInjector.prototype = {
   write: function(outputPath) {
     let inputHTML = fs.readFileSync(path.join(this.rootPath, 'index.html'), 'utf-8');
 
-    this.filePathsToInject.forEach((p) => {
-      if (inputHTML.indexOf(p) === -1) { return; }
+    let generatedCss = [];
+    this.filePathsToInject.forEach((filePath) => {
+      if (filePath) {
```

Comment:
When is a falsy value expected in this list such that you need to guard for it?

Reply:
@mixonic Looks like the output path for vendor css was incorrect.(`this.app.options.outputPaths.app.css.vendor` --> `this.app.options.outputPaths.vendor.css`) Have fixed it now. 👍